luigi/worker.py

```diff
@@ -392,6 +420,10 @@ class worker(Config):
                                           description='If true, use multiprocessing also when '
                                           'running with 1 worker')
 
+    receive_messages = BoolParameter(default=True,
```

Comment:
Maybe `can_receive_messages`?

Also I'm thinking of having the default to `False`, so the UI button doesn't appear by default.